pkg/imgpkg/cmd/describe.go

```diff
@@ -138,14 +141,16 @@ func (p bundleTextPrinter) printerRec(description v1.Description, originalLogger
 		indentLogger.Logf("- Image: %s\n", b.Image)
 		indentLogger.Logf("  Type: Bundle\n")
 		indentLogger.Logf("  Origin: %s\n", b.Origin)
-		indentLogger.Logf("  Layers:\n")
-		for _, d := range b.Layers {
-			indentLogger.Logf("    - Digest: %s\n", d.Digest)
+		if layers {
```

Comment:
instead of having a variable present can we jus change this if to check on `len(b.Layers)`?